script.js

```diff
@@ -1,12 +1,298 @@
 //You can edit ALL of the code here
+//Global variables
+  const search = document.getElementById("search");
+  const searchResult = document.getElementById("searchResult");
+  const rootElem = document.getElementById("root");
+  const selector = document.getElementById("episodeSelector");
+  const showSelect = document.getElementById("showSelector");
+  const shows = getAllShows();
+  // const allEpisodes = getAllEpisodes();
+  let newList = [];
+  let selectedOpt = [];
+  let apiArray; // array to store data fetched
+  let sortedShowList = [];
+  let showDisplayBtn = document.getElementById("display-shows");
+
```

Comment:
Hi Ryan I used to also declare my variables like this but I have read in our code guide that it is bad practice as we should keep our code together, if variable is used in a function declare it there.